src/qz/installer/certificate/firefox/FirefoxCertificateInstaller.java

```diff
@@ -155,4 +164,51 @@ public static void installPolicy(AppLocator app, X509Certificate cert) {
     public static boolean checkRunning(AppLocator app, boolean isSilent) {
         throw new UnsupportedOperationException();
     }
+
+    public static void restartFirefox(ArrayList<AppLocator> appList) throws IOException {
```

Comment:
Costmetic: Consider a new name since we're not actually restarting Firefox any longer.